### PR TITLE
*Rough* fix for appeal challenges my activity

### PR DESCRIPTION
**Pull request by nickreynolds**
- adds appeal challenges to rescue and claim rewards for to my activity
- these do not necessarily render properly, the whole my activity section will be overhauled this sprint, but it does unblock users who want to transfer tokens out of voting contract

### CODE REVIEW

File: packages/dapp/src/components/Dashboard/ActivityList.tsx [hunk 1]
```diff
@@ -37,12 +42,38 @@ class ActivityList extends React.Component<ActivityListOwnProps> {
...
           this.props.challenges.map(c => {
             index++;
             return <ChallengeActivityListItem key={c} challengeID={c!} even={index % 2 === 0} />;
           })}
+        {!this.props.resolvedChallenges &&
+          this.props.appealChallenges &&
+          this.props.appealChallenges.map(c => {
+            index++;
+            return (
+              <ResolvedAppealChallengeActivityListItem
```
Comment (am9u) [inline packages/dapp/src/components/Dashboard/ActivityList.tsx]: Should this be `AppealChallengeActivityListItem`?

File: packages/dapp/src/components/Dashboard/ActivityList.tsx [hunk 2]
```diff
@@ -2,10 +2,15 @@ import * as React from "react";
 import { Set } from "immutable";
 import BigNumber from "bignumber.js";
 import { ChallengeActivityListItem, ActivityListItem, ResolvedChallengeActivityListItem } from "./ActivityListItem";
+import {
+  ResolvedAppealChallengeActivityListItem,
+  // AppealChallengeActivityListItem,
```
Comment (am9u) [inline packages/dapp/src/components/Dashboard/ActivityList.tsx]: Is this import intentionally commented out?

File: packages/dapp/src/components/Dashboard/ActivityList.tsx (hunk 1 above)
Comment (nickreynolds) [reply packages/dapp/src/components/Dashboard/ActivityList.tsx]: as far as I can tell the only difference between the 2 components is that one supports the check-box. I think that we will end up removing the "Resolved" version of both components during this sprint's refactor but wanted to leave it in for now to sorta line up with the non-appeal-challenge versions

File: packages/dapp/src/components/Dashboard/ActivityList.tsx (hunk 2 above)
Comment (nickreynolds) [reply packages/dapp/src/components/Dashboard/ActivityList.tsx]: yeah just to remind us to take a look at it during refactor

**Pull request merged**